### PR TITLE
Routing

### PR DESCRIPTION
**Pull request by hoangtv2605**
ready

### CODE REVIEW

**Comment by hoangtv2605** [comment]
ready

**Comment by hoangminhuet** [comment]
2 commit, gộp commit đi a

File: quickstart/routes/web.php [hunk 1]
```diff
@@ -16,3 +16,5 @@
 Route::get('/', function () {
     return view('welcome');
 });
```
Comment (hoangminhuet) [inline quickstart/routes/web.php]: k viết function trong route

**Review by huyenttm-0901 — COMMENTED** [review]
2 commits, 89 files changed

**Pull request closed without merging**